docs/core/tutorials/publishing-with-visual-studio.md

```diff
@@ -1,49 +1,67 @@
+         This is the application's runtime dependencies file. It defines the .NET Core components and the libraries (including the dynamic link library that contains your application) needed to run the app. For more information, see [Runtime configuration files](https://github.com/dotnet/cli/blob/85ca206d84633d658d7363894c4ea9d59e515c1a/Documentation/specs/runtime-configuration-file.md).
 
       * *HelloWorld.dll*
 
-         The file that contains your application. It is a dynamic link library that can be executed by entering the `dotnet HelloWorld.dll` command in a console window.
+         This is the [framework-dependent deployment](../deploying/deploy-with-cli.md#framework-dependent-deployment) version of the application. To execute this dynamic link library, enter `dotnet HelloWorld.dll` at a command prompt.
+
+      * *HelloWorld.exe*
+      
+         This is the [framework-dependent executable](../deploying/deploy-with-cli.md#framework-dependent-executable) version of the application. To run it, enter `HelloWorld.exe` at a command prompt.
 
       * *HelloWorld.pdb* (optional for deployment)
 
-         A file that contains debug symbols. You aren't required to deploy this file along with your application, although you should save it in the event that you need to debug the published version of your application.
+         This is the debug symbols. You aren't required to deploy this file along with your application, although you should save it in the event that you need to debug the published version of your application.
 
       * *HelloWorld.runtimeconfig.json*
 
-         The application's runtime configuration file. It identifies the version of .NET Core that your application was built to run on. For more information, see [Runtime Configuration Files](https://github.com/dotnet/cli/blob/85ca206d84633d658d7363894c4ea9d59e515c1a/Documentation/specs/runtime-configuration-file.md).
+         This is the application's runtime configuration file. It identifies the version of .NET Core that your application was built to run on. For more information, see [Runtime configuration files](https://github.com/dotnet/cli/blob/85ca206d84633d658d7363894c4ea9d59e515c1a/Documentation/specs/runtime-configuration-file.md).
 
    ![Console window showing published files](media/publishing-with-visual-studio/published-files-output.png)
 
-The publishing process creates a framework-dependent deployment, which is a type of deployment where the published application will run on any platform supported by .NET Core with .NET Core installed on the system. Users can run your application by issuing the `dotnet HelloWorld.dll` command from a console window.
+## See also
```

Comment:
Per the [tutorial template](https://review.docs.microsoft.com/en-us/help/contribute/templates/dotnet/tutorial-template-dotnet?branch=master), this should be "Additional resources"

Reply:
Thanks, I didn't know there was a separate .NET tutorial template.